Microsoft.Toolkit.Uwp.UI.Controls.Design/Controls/TextToolbar.Typedata.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Design
+{
+    internal static partial class ControlTypes
+    {
+        internal const string TextToolbar = RootNamespace + "." + nameof(TextToolbar);
```

Comment:
I'm not a huge fan of this pattern of RootNamespace + nameof.
Do these need to be constants? If not, then I would get typeof(class).FullName. and remove the ".Design" string in the middle.

Reply:
https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3318#discussion_r587616818